Assets/GoogleVR/Plugins/iOS/DaydreamControllerPlugin.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 Google Inc. All rights reserved.
```

Comment:
I stuck with 2017, since that's what the rest of the project is currently using.